schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -916,7 +916,73 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/autoProvisioningSettings"},
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/advancedThreatProtectionSettings"},
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/informationProtectionPolicies"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" }, [](start = 18, length = 127)

I think we don't want to add to the existing root schemas. The new schemas are only for export template, and we are to create a new root schema for that.

Reply:
Updated to create new root schema. Please take another look.